Unreal Engine plugin docs update (#13288)
getsentry/sentry-docs: docs/platforms/unreal/configuration/options.mdx

<Alert>

Attaching screenshots to crash events is currently not supported on Android.

</Alert>

</ConfigKey>

<ConfigKey name="attach-game-log">

> When enabled, game log file is automatically attached to all events captured if the current build configuration allows logging.

Reviewer: ```suggestion
When enabled, and when the current build configuration supports logging, the game's log file is automatically attached to all events.
```

This option is turned off by default.

</ConfigKey>

<ConfigKey name="attach-gpu-dump">

> When enabled, [Nsight Aftermath](https://developer.nvidia.com/nsight-aftermath) mini-dump file is automatically attached to GPU crash events captured.

Reviewer: ```suggestion
When enabled, a [Nsight Aftermath](https://developer.nvidia.com/nsight-aftermath) minidump file is automatically attached to captured GPU crash events.
```

This option is turned on by default.

<Alert>

> This feature is currently supported only for Nvidia GPUs.

Reviewer: ```suggestion
This feature is currently limited to Nvidia GPUs.
```

</ConfigKey>

<ConfigKey name="attach-game-log">

> When enabled, game log filr is automatically attached to all events captured if current build configuration allows logging.

Reviewer 2: filr -> file

These options can be used to hook the SDK in various ways to customize the reporting of events.

> The callbacks you set as hooks will be called on the thread where the event happened. If the event occurs on a non-game thread during garbage collection the callback will not be invoked.

Reviewer: ```suggestion
The hooks you set as callbacks will be called and executed on the same thread where the event happened. Note that if the event occurs on a non-game thread during garbage collection then then callback will not be invoked.
```
